libr/core/cmd_print.c

```diff
@@ -3489,7 +3489,7 @@ static void cmd_print_bars(RCore *core, const char *input) {
 					break;
 				case 'z':
 					if ((IS_PRINTABLE (p[j]))) {
-						if (p[j + 1] == 0) {
+						if (!(&p[j + 1])) {
```

Comment:
wtf this cant be null how to repro this because the fix looks wrong to me

Reply:
you gave me this error and i wrote how to repro in description.